samples/datatypes.js

```diff
@@ -611,6 +607,97 @@ async function queryWithTimestamp(instanceId, databaseId, projectId) {
   // [END spanner_query_with_timestamp_parameter]
 }
 
+async function addNumericColumn(instanceId, databaseId, projectId) {
```

Comment:
For all new samples, we're really trying to keep to one snippet per file:
https://g3doc.corp.google.com/company/teams/cloud-devrel/dpe/samples/nodejs.md?cl=head

Reply:
Ah yes, that makes sense. I've now moved the new samples into their own files.